docs/src/solutions.md

```diff
@@ -268,7 +269,7 @@ For instance, this is how you can use this functionality:
 
 ```julia
 using JuMP
-model = Model()
+model = Model() # With a solver that supports conflicts, like CPLEX or Gurobi
```

Comment:
I'd be more explicit, like "You must use a solver that supports conflicts like CPLEX or Gurobi."

Reply:
Better this way? I also added the IIS keyword there.